src/en/nyxscans/src/eu/kanade/tachiyomi/extension/en/nyxscans/NyxScans.kt

```diff
@@ -0,0 +1,39 @@
+package eu.kanade.tachiyomi.extension.en.nyxscans
+
+import eu.kanade.tachiyomi.multisrc.iken.Iken
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import okhttp3.Response
+
+class NyxScans : Iken(
+    "Nyx Scans",
+    "en",
+    "https://nyxscans.com",
+) {
+    // ============================== Popular ===============================
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        val data = response.asJsoup().getNextJson("popularPosts")
+
+        val entries = json.decodeFromString<List<PopularParseDto>>(data).map { entry ->
```

Comment:
```suggestion
        val entries = data.parseAs<List<PopularParseDto>>.map { entry ->
```
 `parseAs` from utils